pixart.js

```diff
@@ -1 +1,21 @@
+var colorField = $("#color-field")
+var setColor = $("#set-color")
 
+$(document).ready(function(){
+
+  setColor.on("click",function(evt){
+    evt.preventDefault()
+    color = colorField.val()
+    $(".brush").css("background", color)
+  })
+
+  for(i = 0; i<8000; i++) {
+    $("body").append($("<div class='square'></div>"))
+  }
+
+  $(".square").on("mouseover", function(){
+    color = colorField.val()
```

Comment:
Is there a way you could change this line so that the app will let the user draw with the default color as soon as the page loads?